packages/frontend/src/components/Strategies/Bull/BullTrade/Deposit.tsx

```diff
@@ -163,15 +163,14 @@ const BullDeposit: React.FC<{ onTxnConfirm: (txn: BullTransactionConfirmation) =
     const squeethPrice = quote.ethOutForSqth.div(quote.oSqthIn)
     const scalingFactor = new BigNumber(INDEX_SCALE)
     const fundingPeriod = new BigNumber(FUNDING_PERIOD).div(YEAR)
-    const executionVol = new BigNumber(
-      Math.log(scalingFactor.times(squeethPrice).div(normFactor.times(ethIndexPrice)).toNumber()),
-    )
-      .div(fundingPeriod)
-      .sqrt()
-    const showPriceImpactWarning = executionVol
-      .minus(impliedVol)
-      .abs()
-      .gt(BigNumber.max(new BigNumber(impliedVol).times(VOL_PERCENT_SCALAR), VOL_PERCENT_FIXED))
+    const log = Math.log(scalingFactor.times(squeethPrice).div(normFactor.times(ethIndexPrice)).toNumber())
+    const executionVol = new BigNumber(log).div(fundingPeriod).sqrt()
+    const showPriceImpactWarning =
```

Comment:
I actually don't think we need this warning for the withdraw case for log<0 because this is a user buying oSQTH we are most worried about a user selling down to an undefined vol.  We still need it for the normal execution vol case (log >0 case)

If their execution vol is still NaN, that means they are buying when it is super cheap/NaN vol and its actually probably a good trade.

Reply:
This change is only being made for deposit cases.